src/main/java/at/hannibal2/skyhanni/features/combat/damageindicator/MobFinder.kt

```diff
@@ -579,7 +579,7 @@ class MobFinder {
 
             // F5
             "§c[BOSS] Livid§r§f: This Orb you see, is Thorn, or what is left of him." -> {
-                floor5lividEntity = DungeonLividFinder.lividEntity
+                floor5lividEntity = DungeonLividFinder.livid?.baseEntity as? EntityOtherPlayerMP?
```

Comment:
why is here not DungeonLividFinder.lividEntity getting used?

Reply:
answered in the other reply